guides/v2.1/javascript-dev-guide/widgets/widget_menu.md

```diff
@@ -40,6 +41,14 @@ Setting the default responsive handler for the navigation widget.
 
 **Default value**: `false`
 
+### `mediaBreakpoint` {#m_mediaBreakpoint}
+Sets the breakpoint where menu changes approach on how menu is shown and reacts
```

Comment:
This needs a little clarification. Consider the following rewrite:

> Sets the width of user’s window in pixels for which the menu switches between mobile view and desktop view.

or

> Set of options that can be dynamically set while the page is resizing.

Don't forget to make the same changes to the 2.3 file.